src/components/Iwd/IwdThanksPageVariations.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+	<div class="tw-bg-secondary">
+		<div class="row page-content !tw-pt-0">
+			<template v-if="iwdValetInviterId">
+				<div
+					class="
+						tw-flex
+						tw-bg-white
+						tw-shadow-lg
+						tw-mt-0
+						md:tw-mt-2
+						tw-rounded-b
+						md:tw-rounded-t
+						tw-p-1.5
+						tw-gap-1
+						tw-items-center
+					"
+				>
+					<div class="tw-shrink-0">
+						<img :src="iWD2024Badge" alt="IWD Badge" class="tw-w-10 md:tw-w-6">
```

Comment:
Rounded corners?

Reply:
Are you referring to here (the badge) or the borrower image? The mocks didn't show rounded corners, but you are correct that we usually round our borrower images...

Reply:
Ya, the borrower image looks a like clunky + unadorned...

Reply:
Agreed, added rounded corners to match other instances of the borrower image:

![image](https://github.com/kiva/ui/assets/16867161/7485848c-c8ec-4e2c-a292-b1dcdf16b054)